app/api/get-random-whys.js

```diff
@@ -0,0 +1,44 @@
+// https://github.com/EnCiv/civil-pursuit/issues/135
+
+const Points = require('../models/points')
+const { ObjectId } = require('mongodb')
+
+async function getRandomWhys(pointId, category, qty, cb) {
+  if (!this.synuser || !this.synuser.id) {
+    console.error('getRandomWhys called but no user logged in')
+    return cb && cb(undefined)
+  }
+
+  if (!pointId || !category || !qty || typeof qty !== 'number') {
+    console.error('getRandomWhys called with missing or invalid parameters')
+    return cb && cb(undefined)
+  }
+
+  try {
+    // Ensure pointId is converted to ObjectId if it's not already
+    if (typeof pointId === 'string') {
+      pointId = new ObjectId(pointId)
```

Comment:
Points.ObjectId should work here, and then you wouldn't have to require mongodb - It's not important but since I'm asking for the other change, might as well do this.